A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -401,4 +416,78 @@ class A32NX_FWC {
         }
         return true;
     }
+
+    _autopilotDisconnect(_deltaTime) {
+        const apStatus = SimVar.GetSimVarValue("AUTOPILOT MASTER", "Bool");
+        const atherStatus = SimVar.GetSimVarValue("AUTOTHROTTLE ACTIVE", "Bool");
```

Comment:
What's `ather`? Do you mean `athr`, or `autothrottle`?

Reply:
Yes i'll fix it